src/library_html5.js

```diff
@@ -2673,12 +2673,35 @@ var LibraryJSEvents = {
     return requestAnimationFrame(tick);
   },
 
+  // Just like setImmediate but returns an i32 that can be passed back
+  // to wasm rather than a JS object.
+  setImmediateWrapped: function(func) {
```

Comment:
```suggestion
  $setImmediateWrapped: function(func) {
```
And also below, or was there a reason not to?